fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -433,8 +435,8 @@ public void delete() {
 
             node.remove();
 
-            if (parent != null) {
-                createTombstone(parent, name);
+            if (!isTimeMap && parent != null) {
+                    createTombstone(parent, name);
```

Comment:
It looks like an inadvertent tab was added here.

Reply:
Right. I'll remove it. Thanks.